tsconfig.json

```diff
@@ -7,7 +7,9 @@
       "dom"
     ],
     "types": [
-      "node"
+      "node",
+      "mocha",
+      "chai"
```

Comment:
We aren't using `chai` in the global scope, so I don't think we need this.

Reply:
Have removed it from here and the top level `tsconfig.json`.